{{cookiecutter.app_name}}/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "{{cookiecutter.app_name}}",
+  "version": "1.0.0",
+  "description": "{{cookiecutter.project_short_description}}",
+  "main": "js/main.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "NODE_ENV=prod ./node_modules/.bin/webpack --progress --colors -p",
```

Comment:
Not sure it matters, but I've always seen `NODE_ENV=production`. Are you sure that `NODE_ENV=prod` works too?